vehicle/lgsvl_interface/param/lgsvl.param.yaml

```diff
@@ -0,0 +1,35 @@
+# based on vehicle_interface.param.template.yaml
+/**:
+  ros__parameters:
+    cycle_time_ms: 10
+    # Only one of the four control command topics need be specified
+    # "raw", "basic", "ackermann", or "high_level"
+    control_command: "raw"
+    lgsvl:
+      use_nav_odometry_topic: True
+      throttle:
+        domain: [0.0, 3.0]
+        range: [0.0, 100.0]
+      brake:
+        domain: [-3.0, 0.0]
+        range: [100.0, 0.0]
+      steer:
+        domain: [-0.331, 0.331]
+        range: [-100.0, 100.0]
```

Comment:
@TakaHoribe-san, you can see here the mappings that can be configured for the interface.
Complicated mappings are possible for example:
```yaml
        domain: [0.0, 1.0, 1.5, 2.0, 4.0]
        range: [0.0, 10.0, 20.0, 50.0, 100.0]
```

Reply:
Sorry, I missed your coment. Thank you for the information!

Reply:
I'm not sure how the LGSVL mapping is complicated, but this mapping usually depends on its velocity.